filecoin-proofs/src/constants.rs

```diff
@@ -192,6 +194,19 @@ fn parameter_id(cache_id: &str) -> String {
     )
 }
 
+/// Get the correct verifying key data for a given cache id.
+pub fn get_verifying_key_data(cache_id: &str) -> Option<&ParameterData> {
+    PARAMETERS.get(&verifying_key_id(cache_id))
+}
+
+fn verifying_key_id(cache_id: &str) -> String {
+    format!(
+        "v{}-{}.vk",
```

Comment:
Does this format need to be consistent with one from elsewhere? If so, a comment noting where so they can be kept in sync (or a refactor to enforce it) might be helpful.

Reply:
No, nothing is exposed how it's needed here, but could be part of a future re-factor.